core/src/test/java/io/github/belgif/rest/guide/validator/core/model/MediaTypeTest.java

```diff
@@ -0,0 +1,64 @@
+package io.github.belgif.rest.guide.validator.core.model;
+
+import io.github.belgif.rest.guide.validator.core.model.helper.MediaType;
+import io.github.belgif.rest.guide.validator.core.model.helper.MediaTypeException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MediaTypeTest {
+
+    @Test
+    void testIncludes() {
```

Comment:
Too many tests in one test method. Ideally, one test case = one test method so they're independently executed. But to avoid verbosity, you can limit to test methods per "var mediaType = ....".